### ISSUE TITLE
Are the VCRT forwarders really needed?

### ISSUE THREAD

**Issue by sylveon** [windows-toolkit/Microsoft.Toolkit.Win32#150]
## I'm submitting a...
- Feature request

## Current behavior
Currently to use the Win32 XamlHost against a desktop application, we need to carry around the Microsoft.VCRTForwarders package because the host is linked against the app CRT.

## Expected behavior
Considering that VCRTForwarders only forward to the desktop CRT with no additional behavior, wouldn't it be a better idea to link against it directly to reduce the number of files a program needs to carry around? The new Terminal does that already by adding those two properties in the vcxproj file, but still needs the forwarders because of XamlHost:
```xml
<_NoWinAPIFamilyApp>true</_NoWinAPIFamilyApp>
<_VC_Target_Library_Platform>Desktop</_VC_Target_Library_Platform>
```

**Comment by michael-hawker** [comment]
@ocalvo any thoughts on this?

**Comment by ocalvo** [comment]
Love it, we should remove it if we can.

**Comment by sylveon** [comment]
My bad on that part then. At least for my app, the system controls are enough, so linking XamlHost against the desktop CRT will allow me to ship my app without the VCRT forwarders.

**Comment by vitek-karas** [comment]
I would like to add that for .NET Core the forwarders should not be needed at all - .NET Core runtime already contains them on platforms where it's necessary. I haven't really tried all the cases, but I think it should work just fine without them on .NET Core.
Also see https://github.com/dotnet/core-setup/issues/7746 which is indirectly caused by this (and the fact that the VCRT package is not exactly right).

**Comment by michael-hawker** [comment]
Won't fix for 6.0, but will look at again later possibly.

**Comment by michael-hawker** [comment]
@ocalvo @marb2000 I feel like this issue hasn't been resolved fully as there's two proposed PRs by @sylveon to address other open issue we have caused by this dependency.

With the updates to .NET Core has that changed the story here at all?

Wondering if @DHowett @zadjii-msft have any insights here from the Terminal team. (FYI @azchohfi too).
